File: solutions/python/03/graph_adj_list/main.py
def main() -> None:
    n = 5
    g = [[] for _ in range(n)]
    for u, v in [(0, 1), (0, 2), (1, 3), (2, 4)]:
        g[u].append(v)
        g[v].append(u)
    print(g)


if __name__ == "__main__":
    main()

File: solutions/go/03/graph_adj_list/main.go
package main

import "fmt"

func main() {
	n := 5
	g := make([][]int, n)
	edges := [][2]int{{0, 1}, {0, 2}, {1, 3}, {2, 4}}
	for _, e := range edges {
		u, v := e[0], e[1]
		g[u] = append(g[u], v)
		g[v] = append(g[v], u)
	}
	fmt.Println(g)
}
